tests/conftest.py

```diff
@@ -2838,6 +2838,8 @@ def bobby_columnar_table_multi_batch(empty_data_context):
                         "metric_configuration": {
                             "metric_name": "table.row_count",
                             "domain_kwargs": {},
+                            "metric_value_kwargs": None,
+                            "metric_dependencies": None,
```

Comment:
These are parameter that are now outputted by the `profiler`. Was not caught by our previous test, because of how we were doing `sort()`